site2/website/versioned_docs/version-2.2.0/sql-deployment-configurations.md

```diff
@@ -16,7 +16,7 @@ The configurations for the connector and its default values are discribed below.
 connector.name=pulsar
 
 # the url of Pulsar broker service
-pulsar.broker-service-url=http://localhost:8080
+pulsar.web-service-url=http://localhost:8080
```

Comment:
I think we don't need to change the documentation of 2.2.0 version right?

Reply:
Sure